src/app/main/main.component.css

```diff
@@ -0,0 +1,97 @@
+.ads-container {
+    display: grid;
+    -ms-grid-row-align: center;
+    grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+    object-fit: cover;
+    width: 75vw;
+    padding: 5vh 0;
+} 
+
+.ad {
+    position: relative;
+    align-self: center;
+    justify-self: center;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    width: 300px;
```

Comment:
please avoid static sizes

Reply:
I feel 300px for ad is optimized size cause there are not many devices with width smaller than 300-350px